python/paddle/fluid/incubate/fleet/parameter_server/__init__.py

```diff
@@ -227,19 +227,49 @@ def is_server(self):
         """
         return self._role_maker._is_server()
 
-    def init_pserver_model(self):
+    def load_pserver_model(self, load_path, mode=0):
         """
-        init pserver model called from pserver
+        load pserver model called from pserver
+        when you run distributed, use hdfs/afs save_path instead of local
+
+        Args:
+            load_path(str): can be local or hdfs/afs path
+            mode(int): 0 means load all, 1 means load delta, default is 0
+
+        Example:
+            >>> fleet.init_worker(train_program)
+            >>> fleet.load_pserver_model("hdfs:/my/path/to/model/")
+            >>> exe.train_from_dataset(train_program, dataset, debug=False)
         """
+        if mode != 0 and mode != 1:
+            raise ValueError("mode must be 0 or 1, but mode=%s" % mode)
+        mode = str(mode)
+        self._role_maker._barrier_worker()
         if self._role_maker._is_first_worker():
-            self._fleet_ptr.init_model()
+            self._fleet_ptr.load_model(load_path, mode)
         self._role_maker._barrier_worker()
 
-    def save_pserver_model(self, save_path):
+    def save_pserver_model(self, save_path, mode=0):
```

Comment:
the argument of mode needs very detailed comments to clarify the difference between save all and save delta